src/relax/op/tensor/manipulate.h

```diff
@@ -133,6 +133,33 @@ Expr collapse_sum_like(Expr data, Expr collapse_target);
  */
 Expr collapse_sum_to(Expr data, Expr shape);
 
+/*!
+ * \brief Repeats elements of an array.
+ * \param data The input tensor.
+ * \param repeats The number of repetitions.
+ * \param axis The axis along which to repeat values. The negative numbers are interpreted counting
+ * from the backward. By default, use the flattened input array, and return a flat output array.
+ * \return The computed result.
+ */
+Expr repeat(Expr data, PrimExpr repeats, Optional<Integer> axis = NullOpt);
```

Comment:
Seems there is no need to set the default value `= NullOpt`

Reply:
If we use these operators in C++, would providing a default value more convenient?

Reply:
In this case maybe we can add a default value for `repeat`. And also add for other ops with such Optional argument.